Resources/Prototypes/SS220/Catalog/uplink_catalog.yml

```
@@ -30,9 +30,9 @@
   productEntity: MindSlaveImplanter
   discountCategory: veryRareDiscounts
   discountDownTo:
-    Telecrystal: 8
+    Telecrystal: 10 #ss220-uplink-discounts
```

Это не стоит изменять, так как это уже ранее занерфленный билд через имплант + контролер, что берут довольно редко. Суть изменения не ясна, убьёт предмет окончательно скорее всего.
```suggestion
    Telecrystal: 8
```